From 1e2220e132745a4d0e27c647056c93848b189ddb Mon Sep 17 00:00:00 2001
From: Ken Zhu <guigenz@codeaurora.org>
Date: Wed, 28 Apr 2021 09:11:11 -0700
Subject: [PATCH 149/500] netfiter: remove warning of confirmed checking

When bridge-nf-call-iptables is enabled, a multicast
packet could go through both bridging and routing
forwarding process in some cases.
It gets confirmed in bridging forwarding first and then
it run into the warning when reaching nat_setup_info..

Change-Id: Id4b49b81d9748033cc61acea9e4ed96aa2e50c8c
Signed-off-by: Ken Zhu <guigenz@codeaurora.org>
---
 net/netfilter/nf_conntrack_core.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/net/netfilter/nf_conntrack_core.c b/net/netfilter/nf_conntrack_core.c
index 9f6f2e643575..349dfc777de5 100644
--- a/net/netfilter/nf_conntrack_core.c
+++ b/net/netfilter/nf_conntrack_core.c
@@ -2049,9 +2049,6 @@ void nf_conntrack_alter_reply(struct nf_conn *ct,
 {
 	struct nf_conn_help *help = nfct_help(ct);
 
-	/* Should be unconfirmed, so not in hash table yet */
-	WARN_ON(nf_ct_is_confirmed(ct));
-
 	nf_ct_dump_tuple(newreply);
 
 	ct->tuplehash[IP_CT_DIR_REPLY].tuple = *newreply;
-- 
2.34.1

